src/index.ts

```diff
@@ -1,53 +1,48 @@
 import { ApiPromise, WsProvider } from "@polkadot/api";
-import { Header, SignedBlock } from "@polkadot/types/interfaces";
 import "@polkadot/api-augment";
 import * as PromClient from "prom-client"
 import * as http from "http";
 import { config } from "dotenv";
 import BN from "bn.js";
-import { xxhashAsHex } from "@polkadot/util-crypto";
-import * as winston from 'winston'
-import { AccountId, Balance, } from "@polkadot/types/interfaces/runtime"
-import { PalletBagsListListNode, PalletBagsListListBag } from "@polkadot/types/lookup"
 import { ApiDecoration } from "@polkadot/api/types";
+import parachains from "./parachains.json";
+import parachainsids from "./parachains-ids.json";
+import { exit } from "process";
+import {SystemExporter} from "./exporters/system";
+import {BalancesExporter} from "./exporters/balances";
+import {XCMTransfersExporter} from "./exporters/xcmTransfers";
+import {StakingMinerAccountExporter} from "./exporters/stakingMinerAccount";
+import {TransactionPaymentExporter} from "./exporters/transactionPayment";
+import {StakingExporter} from "./exporters/staking";
+import {PalletsMethodsExporter} from "./exporters/palletsMethodsCalls";
+import {logger} from "./logger";
 
 config();
 
-// 15 mins, instead of the default 1min.
-const DEFAULT_TIMEOUT = 15 * 60 * 1000;
+// 30 mins, instead of the default 1min.
+const DEFAULT_TIMEOUT = 30 * 60 * 1000;
 
-export const logger = winston.createLogger({
-	level: process.env.LOG_LEVEL || 'debug',
-	format: winston.format.combine(
-		winston.format.colorize(),
-		winston.format.colorize({
-		}),
-		winston.format.timestamp({
-			format: 'YY-MM-DD HH:MM:SS'
-		}),
-		winston.format.printf(
-			(info) => `[${info.timestamp}] ${info.level}: ${info.message}`
-		)
-	),
-	transports: [new winston.transports.Console()]
-})
+export function getParachainName(mykey: string): string {
+	for (const [key, value] of Object.entries(parachainsids)) {
```

Comment:
```suggestion
	for (const [key, value] of Object.entries(parachainsIds)) {
```

Typical JS/Ts code follows camelCase naming.